### PR TITLE
plugin.ts install test case complete

### PR DESCRIPTION
**Pull request by ZijinGao**
I completed the test case for install. Added two cases: when the version is like @1.0.1 and installation is executed; when the version is like @1.0.1 and installation is not executed.

### CODE REVIEW

File: packages/cli/src/bin/pipcook.test.ts [hunk 1]
```diff
@@ -25,3 +25,8 @@ test.serial('fetch with cache', async (t) => {
   t.deepEqual(stubMkdirp.args[0], [ opts.output ] as any, 'should make the currect directory');
   t.true(stubRun.calledOnce, 'run should be called once');
 });
+
+
+// ava, log
+// yarn cov, log
+// .nycrc 配置/pipcook.ts位置/自执行function/。。
```
Comment (yorkie) [inline packages/cli/src/bin/pipcook.test.ts:32]: Remove the above lines.

File: packages/cli/src/utils/plugin.test.ts [hunk 2]
```diff
@@ -40,7 +40,52 @@ test.serial('should install', async (t) => {
   t.true(stubPathExists.calledOnce);
   t.false(stubMkdirp.called);
   t.true(stubExecAsync.calledOnce);
-  t.deepEqual(requirePath, expected);
+  t.is(requirePath, expected);
+});
+
+/* new tc */
```
Comment (yorkie) [inline packages/cli/src/utils/plugin.test.ts:46]: This line is unnecessary.

File: ssd-tf2/network.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
+from tensorflow.keras import Model
```
Comment (yorkie) [inline ssd-tf2/network.py:1]: What does this for?

**Review by yorkie — CHANGES_REQUESTED** [review]

File: ssd-tf2/network.py (hunk 3 above)
Comment (ZijinGao) [reply ssd-tf2/network.py:1]: This is an old file that should be deleted. I'll delete this in the next commit.

**Comment by WenheLI** [comment]
@ZijinGao - Should sign the CLA first

File: packages/cli/src/bin/pipcook.test.ts [hunk 4]
```diff
@@ -24,4 +24,4 @@ test.serial('fetch with cache', async (t) => {
   t.true(stubMkdirp.calledOnce, 'mkdirp should be called once');
   t.deepEqual(stubMkdirp.args[0], [ opts.output ] as any, 'should make the currect directory');
   t.true(stubRun.calledOnce, 'run should be called once');
-});
+});
```
Comment (FeelyChau) [inline packages/cli/src/bin/pipcook.test.ts]: Missing EOF.

File: packages/cli/src/utils/plugin.ts [hunk 5]
```diff
@@ -46,7 +46,7 @@ export const install = async (name: string, pluginHomeDir: string): Promise<stri
   const requirePath = path.join(pluginHomeDir, 'node_modules', alias);
   // always update plugin if version is 'beta', 'alpha' or 'latest'
   if ([ 'beta', 'alpha', 'latest' ].includes(pluginVersion.version) || !(await fs.pathExists(requirePath))) {
-    await execAsync(
+    await execAsync(
```
Comment (FeelyChau) [inline packages/cli/src/utils/plugin.ts]: Space after `(`.

**Review by FeelyChau — CHANGES_REQUESTED** [review]

**Review by FeelyChau — APPROVED** [review]
LGTM if CI is green.

**Review by yorkie — APPROVED** [review]

**Comment by yorkie** [comment]
PIng @rickycao-qy for code review.

**Review by rickycao-qy — APPROVED** [review]
LGTM

**Pull request merged**